lxd/images.go

```diff
@@ -2637,7 +2637,7 @@ func imageValidSecret(s *state.State, r *http.Request, projectName string, finge
 			continue
 		}
 
-		if !shared.StringInSlice(fmt.Sprintf("/1.0/images/%s", fingerprint), opImages) {
+		if !shared.StringInSlice(fmt.Sprintf("/%s/images/%s", version.APIVersion, fingerprint), opImages) {
```

Comment:
Just a thought here, but as we are going to be adding the `?project` param to the URLs in resources. 
Then does this mean that this condition will never match for image resources in the non-default project?

Should we be iterating through the `opImages` slice and doing a prefix match instead?